tsdb/engine/tsm1/reader.go

```diff
@@ -1374,6 +1375,10 @@ func (m *mmapAccessor) init() (*indirectIndex, error) {
 		return nil, fmt.Errorf("mmapAccessor: invalid indexStart")
 	}
 
+	// Hint to the kernal that we will be reading the file.  It would be better to hint
+	// that we will be reading the index section, but that doesn't seem to work ATM.
+	_ = madvise(m.b, syscall.MADV_WILLNEED)
```

Comment:
Could we add this call to the Series File index? We will always need to read from that file too right?

Reply:
I was not measuring series file IOPS so I would not add this there without seeing the actual change in performance.   This does have _some_ benefit with larger TSM files that are not in the page cache though.